scripts/build/docker/run-command.sh

```diff
@@ -82,6 +82,8 @@ fi
 # The `-t` and TERM setup is needed to display coloured output.
 docker run -t \
   --env "TERM=xterm-256color" \
+  --env "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" \
```

Comment:
Should the `RELEASE_TYPE` variable also be transmitted from Travis to Docker using the same mechanism?